firefox_puppeteer/base.py

```diff
@@ -28,16 +28,44 @@ def get_marionette(self):
 class UIBaseLib(BaseLib):
     """A base class for all UI element wrapper classes inside a chrome window."""
 
-    def __init__(self, marionette_getter, window):
-        BaseLib.__init__(self, marionette_getter)
-
+    def __init__(self, marionette_getter, parent, element):
```

Comment:
I think this is confusing if in some cases our "parent" is the parent in the DOM and in some cases it isn't.